CHANGELOG_DEV.md

```diff
@@ -1,3 +1,11 @@
+## 0.91.0 - 2021-12-01
+### Changed
+- wrap with mutex `acra-censor's` query writer's manipulations of cached queries to avoid race conditions
```

Comment:
```suggestion
- wrap `acra-censor`'s query writers' manipulations of cached queries with a mutex to avoid race conditions
```
*unsure whether this is better, but you may consider this variant*